instrumentation/activity/README.md

```diff
@@ -23,15 +23,17 @@ This instrumentation produces the following telemetry:
 ### Activity state change
 
 * Type: Span
-* Name: {`Created` | `Resumed` | `Paused` | `Stopped` | `Destroyed`} (depends on the activity state transition)
+* Name: {`Created` | `Resumed` | `Paused` | `Stopped` | `Destroyed` | `Restarted` |} (depends on the activity state transition)
 * Description: As the activity transitions between states, a span will be created to represent the
   lifecycle of that state. Events are added for subsequent minor state changes.
 * SpanEvents: {
   `activityPreCreated` | `activityCreated` | `activityPostCreated` |
+  `activityPreStarted` | `activityStarted` | `activityPostStarted` |
   `activityPreResumed` | `activityResumed` | `activityPostResumed` |
   `activityPrePaused` | `activityPaused` | `activityPostPaused` |
   `activityPreStopped` | `activityStopped` | `activityPostStopped` |
   `activityPreDestroyed` | `activityDestroyed` | `activityPostDestroyed` }
 * Attributes:
-  * `activity.name`:  <name of activity>
+  * `activityName`:  <name of activity>
```

Comment:
are you sure? see https://github.com/open-telemetry/opentelemetry-android/blob/52c302b93649a90df21a5c814fe020951716fe36/instrumentation/slowrendering/src/main/java/io/opentelemetry/android/instrumentation/slowrendering/SlowRenderListener.java#L202

Reply:
Oh boy. We are inconsistent right now and should fix this. The ActivityTracer uses `activityName` as seen here https://github.com/open-telemetry/opentelemetry-android/blob/main/instrumentation/activity/src/main/java/io/opentelemetry/android/instrumentation/activity/ActivityTracer.java#L26

Reply:
Also, having a semantic convention for this would help..... (hint hint) 😅

Reply:
yeah lets fix this, so they are consistent (and possibly a semantic convention follow up but not a blocker).